cmd/kubectl-gadget/profile/block-io_test.go

```diff
@@ -0,0 +1,138 @@
+package profile
```

Comment:
nit: missing license header.

Reply:
Yes, that's fine, feel free to start the discussion on that. I don't have any strong preference, so I'll ping @vbatts  and @blixtra  on this one.